src/components/FilterSearch.tsx

```diff
@@ -55,6 +55,22 @@ export interface OnSelectParams {
   executeFilterSearch: (query?: string) => Promise<FilterSearchResponse | undefined>
 }
 
+/**
+ * The parameters that are passed into {@link FilterSearchProps.onDropdownInputChange}.
+ *
+ * @public
+ */
+export interface OnDropdownInputChangeProps {
+  /** The new input element's new value after the change */
+  value: string,
+  searchFields: Omit<SearchParameterField, 'fetchEntities'>[],
```

Comment:
is there a reason to have the `searchFields` passed in here? the user passes them to the `FilterSearch` component to begin with, so they should already have that object created

Reply:
The reason I included `searchFields` as a prop for the `onDropdownInputChange` function is so the user can write one function to pass into `onDropdownInputChange` for multiple `FilterSearch` components that may be on one single page. If I don't pass `searchFields`, the user would need to write a unique function for each `FilterSearch` component that uses the proper `searchFields` that belong to that `FilterSearch` instance.